### PR TITLE
Add assortment media

### PR DESCRIPTION
**Pull request by Mikearaya**
Solves #357. basically same functionality available for product media is available here. I know the issue requests for a single page but I don't think it's necessary to add that restriction but let the use case dictate how many images assortments should have. 
even though the meta field will be removed in the coming version, assortment media still support it to comply with API `Media` schema
Major addition is the new collection `assortment_document`.

### CODE REVIEW

File: packages/core-assortments/db/assortment-media/schema.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      authorId: { type: String, required: true },
+      title: String,
+      subtitle: String,
+      ...Schemas.timestampFields,
+    },
+    { requiredByDefault: false }
+  )
+);
+
+Migrations.add({
```
Comment (pozylon) [inline packages/core-assortments/db/assortment-media/schema.js:40]: The migration should not be needed for a completely new feature

File: packages/core-assortments/db/migrations.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+              authorId: 1,
+            },
+          }
+        );
+      });
+  },
+});
+
+export default () => {
+  runAssortmentMediaMigrations();
```
Comment (pozylon) [inline packages/core-assortments/db/migrations.js:68]: what you import here is not migrations, it's index creation? please check this code again

**Review by pozylon — APPROVED** [review]
Be careful when moving files (packages/core-assortments/db/collections.js → ...assortments/db/assortments/collections.js) and extra check for imports for core-assortments and if references still work, search for "core-assortments" in the whole codebase.

**Comment by pozylon** [comment]
Sorry, didn't want to approve, please look at my comments first before you merge

**Comment by Mikearaya** [comment]
Ok, I'll remove migration and rename it appropriatly. as far as moving files I've checked for any unintended issues, but I'll double-check again

**Pull request merged**